public/src/components/epicTests/epicTestVariantEditor.tsx

```diff
@@ -53,7 +53,7 @@ interface Props extends WithStyles<typeof styles> {
   variant?: EpicVariant,
   onVariantChange: (updatedVariant: EpicVariant) => void,
   editMode: boolean,
-  onDelete: (variantName: string) => void
+  onDelete: () => void
```

Comment:
The parent knows the variant's name when it creates this component, so no need to pass the name up

Reply:
Ah!